server/src/server.ts

```diff
@@ -53,17 +53,20 @@ export default class BashServer {
 
   private documents: LSP.TextDocuments<TextDocument> = new LSP.TextDocuments(TextDocument)
   private connection: LSP.Connection
+  private clientCaps: LSP.ClientCapabilities
```

Comment:
Nitpick: but let us spell this out. clientCapabilities.

Reply:
I got it.